pandas/tests/frame/test_indexing.py

```diff
@@ -1944,6 +1944,29 @@ def test_set_value_resize(self, float_frame):
         with pytest.raises(ValueError, match=msg):
             res3._set_value("foobar", "baz", "sam")
 
+    def test_loc_indexing_preserves_index_category_dtype(self):
+        # GH 15166
+        df = DataFrame(
+            data=np.arange(2, 22, 2),
+            index=pd.MultiIndex(
+                levels=[pd.CategoricalIndex(["a", "b"]), range(10)],
+                codes=[[0] * 5 + [1] * 5, range(10)],
+                names=["Index1", "Index2"],
+            ),
+        )
+
+        result_1 = df.index.levels[0]
```

Comment:
can you write this like

expected=

result=
tm.assert_index_equal....

result=
tm.assert_index_equal....

Reply:
Thanks, @jreback - done.